packages/terra-table/src/TableSubheader.jsx

```diff
@@ -19,21 +22,14 @@ const TableSubheader = ({
   content,
   colSpan,
   ...customProps
-}) => {
-  const contentClassName = classNames([
-    'terra-Table-subheader',
-    customProps.className,
-  ]);
-
+}) => (
   // count is based on the number of columns and assigned in the table component which contains this subheader
-  return (
-    <tr className="terra-Table-subheaderRow">
-      <td {...customProps} className={contentClassName} colSpan={colSpan}>
-        {content}
-      </td>
-    </tr>
-  );
-};
+  <tr data-class="subheader-row">
+    <td {...customProps} className={cx('subheader', customProps.className)} colSpan={colSpan}>
```

Comment:
Sorry for unclear question. I wonder what if consumer uses some class name inside `Table.scss`. In that case, our local css style code will be applied or not. For example:
```javascript
// customProps.className includes 'striped' class
<td {...customProps} className={cx('subheader', customProps.className)} .../>
```
In this case, striped style code in [here](https://github.com/cerner/terra-core/blob/8934dc09cbc70d876252a6bbc8ca26d7a0e09e67/packages/terra-table/src/Table.scss#L118) will be applied to `td` or not?